melosys-eessi-app/src/main/java/no/nav/melosys/eessi/controller/dto/Bruker.java

```diff
@@ -21,7 +23,7 @@
     private LocalDate foedseldato;
 
     private String kjoenn;
-    private String statsborgerskap;
+    private List<String> statsborgerskap;
```

Comment:
Jeg synes det er greit å holde typen så generell som mulig. Om det ikke er noen spesifikk grunn til at man ønsker å bruke `List`, så kan vi heller bruker `Collection`. En grunn for å bruke lister er hovedsakelig bare for å bevare rekkefølge og akseptere duplikater. Her bryr vi ikke oss om noen av delene, så da passer Collection best 😄